pylintrc

```diff
@@ -52,7 +52,8 @@ confidence=
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
 
-disable=invalid-name,protected-access,fixme,too-many-branches,
+disable=implicit-str-concat-in-sequence,
```

Comment:
Where exactly do we have this occurring in our code base? It would be nice to see them before merging, since this usually can reveal false positives and whatnot, things we should account for.

Reply:
There are 220 violations :(

And yes, this is because of the way message definitions are built.
For example there is the line where the 1st warning is raised:
https://github.com/PyCQA/pylint/blob/master/pylint/extensions/mccabe.py#L131

Interestingly, it is not consistent, and sometimes the newline is escaped:
https://github.com/PyCQA/pylint/blob/master/pylint/checkers/exceptions.py#L62

What do you advise ?
Replace all string concatenations by escaped newlines ? :(

Reply:
I was afraid this is going to happen with this PR, it will emit even for use cases that are valid, such as the ones from pylint's message definition. Given this, I'm finding  the new check too nitpicky. If we were to include this in pylint, then we should restrict it to verify just the implicit string concatenation that occurs on the same line, not on multiple lines.